config/owasp/suppressions.xml

```diff
@@ -14,4 +14,12 @@
     <notes>This vulnerability exists across azure identity and microsoft authentication libraries </notes>
     <cve>CVE-2024-35255</cve>
   </suppress>
+  <suppress>
```

Comment:
These can now be removed following the dependency update merge

Reply:
removed.